apps/frontend/src/modules/users/slices/users.slice.ts

```diff
@@ -4,34 +4,64 @@ import { DataStatus } from "~/libs/enums/enums.js";
 import { type ValueOf } from "~/libs/types/types.js";
 import { type UserGetAllItemResponseDto } from "~/modules/users/users.js";
 
-import { loadAll, updateProfile } from "./actions.js";
+import { configureGroupUsers, loadAll, updateProfile } from "./actions.js";
 
 type State = {
-	dataStatus: ValueOf<typeof DataStatus>;
+	groupUsers: {
+		data: UserGetAllItemResponseDto[];
+		dataStatus: ValueOf<typeof DataStatus>;
+		totalCount: number;
+	};
 	updateProfileStatus: ValueOf<typeof DataStatus>;
-	users: UserGetAllItemResponseDto[];
-	usersTotalCount: number;
+	users: {
+		data: UserGetAllItemResponseDto[];
+		dataStatus: ValueOf<typeof DataStatus>;
+		totalCount: number;
+	};
 };
 
 const initialState: State = {
-	dataStatus: DataStatus.IDLE,
+	groupUsers: {
+		data: [],
+		dataStatus: DataStatus.IDLE,
+		totalCount: 0,
+	},
 	updateProfileStatus: DataStatus.IDLE,
-	users: [],
-	usersTotalCount: 0,
+	users: {
+		data: [],
+		dataStatus: DataStatus.IDLE,
+		totalCount: 0,
+	},
```

Comment:
reducers should have plain structure, let's return logic with users here back, and move groupUsers as `users` to groups slice because it's related to group creation